cloudsql-postgresql-plugin/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright © 2020 CDAP
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>database-plugins</artifactId>
+    <groupId>io.cdap.plugin</groupId>
+    <version>1.4.0-SNAPSHOT</version>
+  </parent>
+
+  <name>CloudSQL PostgreSQL plugin</name>
+  <artifactId>cloudsql-postgresql-plugin</artifactId>
+  <version>1.4.0-SNAPSHOT</version>
+  <modelVersion>4.0.0</modelVersion>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-etl-api</artifactId>
```

Comment:
the scope should be provided. 
https://github.com/data-integrations/google-cloud/blob/develop/pom.xml#L150

Reply:
Changed scope to provided for cdap-etl-api and hydrator-test.